tests/test_persistence.py

```diff
@@ -52,3 +52,44 @@ def test_libsecret_persistence(temp_location):
         {"my_attr_1": "foo", "my_attr_2": "bar"},
         ))
 
+@pytest.mark.skipif(
+    is_running_on_travis_ci, reason="Requires manual testing")
+def test_windows_empty_file_exists_before_first_use():
+    test_folder = tempfile.mkdtemp(prefix="msal_extension_test_windows_token_cache_roundtrip")
+    cache_file = os.path.join(test_folder, 'msal.cache')
+    open(cache_file, 'w+')
+    try:
+        persistence = FilePersistenceWithDataProtection(cache_file)
+        assert persistence.load() is None  # Empty file should just be a NO-OP
+    finally:
+        shutil.rmtree(test_folder, ignore_errors=True)
+
+@pytest.mark.skipif(
+    is_running_on_travis_ci, reason="Requires manual testing")
+def test_windows_file_contains_data_before_first_use():
```

Comment:
I do not really think this test case is necessary, because this is the default and expected behavior.

If you really want to add it, that's also fine, but please follow the suggestion above.